### DISCUSSION TITLE
Discord invite?

### DISCUSSION THREAD

**Discussion by twoxfh** [gzuidhof/starboard-notebook#103] [Q&A] [answered]
I have never discorded, but given the abundance of requests for invites, can I join the club?

**Comment by unhott** [comment, marked as answer]
https://discord.gg/sKdpcGGc Here's a 7 day link :)

**Comment by twoxfh** [comment]
Thanks!

**Comment by zdmc23** [comment]
Hi, can I get a fresh Discord server link?  Thx!

**Reply by unhott** [reply]
Here you go! https://discord.gg/zvwdKGTQ

**Comment by dit7ya** [comment]
Looking for a new invite link.

**Reply by unhott** [reply]
Here you go https://discord.gg/sXAm46Qf

**Reply by tennox** [reply]
Is it still existent? Would love to be in :sweat_smile:
